src/plugins/intel_cpu/tests/unit/denormals_optimize_test.cpp

```diff
@@ -0,0 +1,191 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include <openvino/core/model.hpp>
+#include <openvino/openvino.hpp>
+#include <openvino/opsets/opset1.hpp>
+#include <openvino/reference/multiply.hpp>
+#include <thread>
+#include <utility>
+
+#include "common_test_utils/graph_comparator.hpp"
+#include "common_test_utils/test_common.hpp"
+#include "utils/denormals.hpp"
+
+using namespace testing;
+
+class DenormalsOptimizeTestF : public ov::test::TestsCommon {
+public:
+    DenormalsOptimizeTestF() {
+        // Default setting
+        ov::intel_cpu::flush_to_zero(false);
+        ov::intel_cpu::denormals_as_zero(false);
+    }
+    ~DenormalsOptimizeTestF() {
+        if (pConst1)
+            delete[] pConst1;
+        if (pConst2)
+            delete[] pConst2;
+        if (pConst3)
+            delete[] pConst3;
+
+        // Recover the default setting to avoid impacting other tests.
+        ov::intel_cpu::flush_to_zero(false);
+        ov::intel_cpu::denormals_as_zero(false);
+    }
+
+    void SetUp() override {
+        initBuf(pConst1, shape_size(inpShape), std::numeric_limits<float>::min());
```

Comment:
Is there any reason to apply float::min?

Reply:
min float * 0.5, we will get less value than min float, so system will use subnormal to express it.
If we set denormal optimization to true, subnormal will be as zero.
Else, system will use subnormal to express this value(less min float).